File: C#/fibonacci.cs
using System;

class FibonacciGenerator
{
    static void Main()
    {
        Console.Write("Enter the number of Fibonacci numbers to generate: ");
        int count = int.Parse(Console.ReadLine());

        Console.WriteLine("Fibonacci Sequence:");
        for (int i = 0; i < count; i++)
        {
            Console.Write(Fibonacci(i) + " ");
        }
    }

    static int Fibonacci(int n)
    {
        if (n <= 1)
            return n;

        return Fibonacci(n - 1) + Fibonacci(n - 2);
    }
}

File: Python/fibonacci.py
# Generate Fibonacci sequence
n = int(input("Enter the number of Fibonacci terms to generate: "))

# Initialize the first two terms
a, b = 0, 1

# Check if the number of terms is valid
if n <= 0:
    print("Please enter a positive integer.")
elif n == 1:
    print("Fibonacci sequence:")
    print(a)
else:
    print("Fibonacci sequence:")
    count = 0
    while count < n:
        print(a)
        next_term = a + b
        a = b
        b = next_term
        count += 1
